src/utilities/__tests__/compareSCO.tests.js

```diff
@@ -0,0 +1,104 @@
+import { compareSCO } from '../compareSCO';
+
+describe('<Compare default state with given search criteria object />', () => {
+	/* eslint-disable testing-library/no-node-access */
+	it('returns true if SCO has not been updated', () => {
+		const searchObject = {
+			age: '', // (a) Age
+			cancerType: { name: '', codes: [] }, // (ct) Cancer Type/Condition
+			subtypes: [], // (st) Subtype
+			stages: [], // (stg) Stage
+			findings: [], // (fin) Side effects
+			keywordPhrases: '', // (q) Cancer Type Keyword (ALSO Keyword Phrases)
+			zip: '', // (z) Zipcode
+			zipCoords: { lat: '', long: '' },
+			zipRadius: '100', //(zp) Radius
+			country: 'United States', // (lcnty) Country
+			states: [], // (lst) State
+			city: '', // (lcty) City
+			hospital: { term: '', termKey: '' }, // (hos) Hospital
+			healthyVolunteers: false, // (hv) Healthy Volunteers,
+			trialTypes: [
+				{ label: 'Treatment', value: 'treatment', checked: false },
+				{ label: 'Prevention', value: 'prevention', checked: false },
+				{ label: 'Supportive Care', value: 'supportive_care', checked: false },
+				{
+					label: 'Health Services Research',
+					value: 'health_services_research',
+					checked: false,
+				},
+				{ label: 'Diagnostic', value: 'diagnostic', checked: false },
+				{ label: 'Screening', value: 'screening', checked: false },
+				{ label: 'Basic Science', value: 'basic_science', checked: false },
+				{ label: 'Other', value: 'other', checked: false },
+			], // (tt) Trial Type
+			trialPhases: [
+				{ label: 'Phase I', value: 'i', checked: false },
+				{ label: 'Phase II', value: 'ii', checked: false },
+				{ label: 'Phase III', value: 'iii', checked: false },
+				{ label: 'Phase IV', value: 'iv', checked: false },
+			], // (tp) Trial phase
+			vaOnly: false, // (va) VA facilities only
+			drugs: [], // (dt) Drug/Drug family
+			treatments: [], // (ti) Treatment/Interventions
+			trialId: '', // (tid) Trial ID,
+			investigator: { term: '', termKey: '' }, // (in) Trial investigators ('in' is legacy but is a keyword and does not work well as a key name; be ready to handle both in query string)
+			leadOrg: { term: '', termKey: '' }, // (lo) Lead Organization
+			resultsPage: 5,
+
+			formType: 'basic', // (empty string (default) | basic | advanced)
+			location: 'search-location-all', // active location option (search-location-all | search-location-zip | search-location-country | search-location-hospital | search-location-nih)
+		};
+
+		expect(compareSCO(searchObject)).toBeTruthy();
+	});
+
+	it('returns false when SCO is different from default state', () => {
+		const searchObject = {
+			age: '40', // (a) Age
```

Comment:
just import the default SCO and spread and modify some of the fields to get a new object

Reply:
done!